src/effects/effect_65.c

```diff
@@ -257,124 +258,134 @@ void fx_65_appendGfx(void* effect) {
     if (sp54 >= 2) {
         gDPSetCombineLERP(gMasterGfxPos++, SHADE, ENVIRONMENT, TEXEL0, ENVIRONMENT, TEXEL0, 0, SHADE, 0, 0, 0, 0, COMBINED, COMBINED, 0, PRIMITIVE, 0);
     }
-    var_fp = 0;
-    sp60 = -1;
-    sp64 = (sp50 & 0x3F) << 5;
-    gDPSetPrimColor(gMasterGfxPos++, 0, 0, data->unk_18, data->unk_1C, data->unk_20, temp_s5);
+
+    gDPSetPrimColor(gMasterGfxPos++, 0, 0, data->unk_18, data->unk_1C, data->unk_20, primAlpha);
     gDPSetEnvColor(gMasterGfxPos++, data->unk_28, data->unk_2C, data->unk_30, 0);
-    gSPBranchList(gMasterGfxPos, &gMasterGfxPos[121]);
+    gSPBranchList(gMasterGfxPos, gMasterGfxPos + 0x79);
```

Comment:
Might be nice to express this 0x79 in terms on which gfx commands are being jumped over. Seems like 29x gSPVertex/gSP2Triangles + gSPPopMatrix + ???